app/javascript/packages/address-search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@18f/identity-address-search",
-  "version": "1.0.5",
+  "version": "2.0.0",
```

Comment:
Is this change expected here? I suppose it shouldn't matter much if it's already in `main` via #9103.